cmd/latest.go

```diff
@@ -30,13 +30,21 @@ func init() {
 		false,
 		"Remove 'v' prefix before echoing",
 	)
+
+	latestCmd.Flags().BoolVarP(
+		&latestSkipPrereleases,
+		"no-prereleases",
```

Comment:
I would flip this around to "prereleases" and have the default operation be the current version. As is, this is a breaking change as it alters the default behavior. We should let users opt-in to the new behavior. If the new behavior is preferred, then in v2 we can swap it.

Reply:
but this is the case and this is what I wanted: the default is non-breaking the current usage- it does not skip pre-releases.
Hence the "negative" flag to change the default behavior.

Reply:
Not done. Misunderstanding about bringing a breaking change.

Reply:
Yea the default would not change but we want the user to opt in and enable the new method. So the parameter would be to add prereleases to the search. So the argument would be --prereleases and it should be false by default. 

If we wanted to disable a feature that was on by default we would set prereleases to false or use the no prefix.

Reply:
Edit: I think I misread something in the issue and the PR and have the above command backwards. Let me re-review this after I get into my office. I think you have it right.